tf/terraform.go

```diff
@@ -238,6 +238,16 @@ func (t *Terraform) Apply() error {
 	return t.exec.Apply(context.TODO())
 }
 
+func (t *Terraform) RefreshState(resources []string) error {
+	opts := make([]tfexec.ApplyOption, 0)
+	// TODO: refresh-only option is not supported in terraform-exec, use tfexec.DirOrPlan as a hack
```

Comment:
Can we simply use `Refresh`?

Reply:
Yes, but `refresh` command is deprecated: https://www.terraform.io/cli/commands/refresh#command-refresh

Reply:
Indeed, whilst probably still better than the hack used here?

Reply:
Good point, I'll change it to refresh command.